core/src/io/Dataparser.cpp

```diff
@@ -1059,32 +1070,31 @@ namespace IO
             myfile.Read_String( ovf_data_representation, "# Begin: Data" );
             std::istringstream repr( ovf_data_representation );
             repr >> ovf_data_representation;
-            if( ovf_data_representation == "Binary" ) 
+            if( ovf_data_representation == "binary" ) 
                 repr >> ovf_binary_length;
             
             Log( lvl, sender, fmt::format( "# OVF data representation = {}", ovf_data_representation ) );
             Log( lvl, sender, fmt::format( "# OVF binary length       = {}", ovf_binary_length ) );
             
             // Check that representation and binary length valures are ok
-            if( ovf_data_representation != "Text" && ovf_data_representation != "Binary" )
+            if( ovf_data_representation != "text" && ovf_data_representation != "binary" )
             {
                 spirit_throw(Utility::Exception_Classifier::Bad_File_Content, Utility::Log_Level::Error,
-                    "Data representation must be either \"Text\" or \"Binary\"");
+                    "Data representation must be either \"text\" or \"binary\"");
             }
             
-            if( ovf_data_representation == "Binary" && 
+            if( ovf_data_representation == "binary" && 
                  ovf_binary_length != 4 && ovf_binary_length != 8  )
             {
                 spirit_throw(Utility::Exception_Classifier::Bad_File_Content, Utility::Log_Level::Error,
-                    "Binary representation can be either \"Binary 8\" or \"Binary 4\"");
+                    "Binary representation can be either \"binary 8\" or \"binary 4\"");

```

Comment:
In all these logging cases, it should be "Binary" and "Text" (i.e. first letter uppercase), as this is the way that MuMax writes it's files.

Reply:
We are following the official OVF 2.0 format. Since we are dealing with capitalization properly this exception will be raised only in case of misspelling of the data type field eg `"txt"` instead of `"text"`in case that someone tries to parse in a human-written OVF file. Of course the choice is up to you.